capstone-frontend/src/components/BookSearchTile.jsx

```diff
@@ -1,14 +1,57 @@
 import React from 'react';
-import '../App.css';
+import { Container, Row, Col, DropdownButton, Dropdown } from 'react-bootstrap';
+import '../styles/BookSearchTile.css';
+
+const addBookToBookList = (bookId, bookListId) => {
+  console.log(`Added book with id: ${bookId} to bookList with id: ${bookListId}`);
+}
+
+const addBookToClub = (bookId, clubId) => {
+  console.log(`Added book with id: ${bookId} to club with id: ${clubId}`);
+}
+
+const addBookToCommunity = (bookId, communityId) => {
+  console.log(`Added book with id: ${bookId} to community with id: ${communityId}`);
+}
 
 const BookSearchTile = (props) => {
   return (
     <div className="book-search-tile">
-      <img className="book-img-med" src={props.thumbnailLink} alt={props.title}/>
-      <div>
-        <div className="book-title"> {props.title} </div>
-        <div className="book-author"> {props.author.join()} </div>
-      </div>
+      <Container>
```

Comment:
These tiles look great!

Reply:
Thanks! Hopefully we can use a similar format for the booklist search tile/club search tile when we get to implementing that in search!